articles/connections/database/custom-db.md

```diff
@@ -143,22 +159,21 @@ For example:
 callback(new ValidationError('email-too-long', 'Email is too long.'));
 ```
 
-## 6. Debug and troubleshoot
+## Troubleshoot
 
 Test the script using the **TRY** button. If your settings are correct you should see the resulting profile:
 
 ![](/media/articles/connections/database/mysql/db-connection-try-ok.png)
 
 If you add a `console.log` statement to the script you will be able to see the output here.
 
-
-## The script container
-
-The script runs in a JavaScript sandbox where you can use the full power of the JavaScript language and selected Node.js libraries. [You can `require` any library listed here](https://tehsis.github.io/webtaskio-canirequire/).
+::: note
+The [auth0-custom-db-testharness library](https://www.npmjs.com/package/auth0-custom-db-testharness) can be used to deploy, execute, and test the output of database action scripts using a Webtask sandbox environment.
+:::
 
 ## Auth0 Login widget
 
-After you have enabled the database connection, the Auth0 Login widget will automatically change its appearance to allow users to enter their `username` and `password`. Once entered, this data is passed to your scripts.
+After you enabling the database connection, the Auth0 Login widget will automatically change its appearance to allow users to enter their `username` and `password`. Once entered, this data is passed to your scripts.
```

Comment:
`after you enabling` does not make sense to me. maybe `after you enable`?

Also this section is applicable only if you use Lock. We should clarify this. Also, I don't get why we format the `username` and `password` as code.